Sources/SwiftPriorityQueue/SwiftPriorityQueue.swift

```diff
@@ -69,6 +73,23 @@ public struct PriorityQueue<T: Comparable> {
         swim(heap.count - 1)
     }
     
+    /// Attempt to add a new element onto an 'inverted' Priority Queue, limiting its size. O(lg n)
+    ///
+    /// If the size limit has been reached and the element falls beyond the 'tail', it will be ignored.  If pushed, a single element at the 'tail' will be popped.
+    ///
+    /// - parameter element: The element to be attempted insertion into the Priority Queue.
+    /// - parameter maxHeap: The Priority Queue will not grow further if its size >= maxHeap.
+    public mutating func push(_ element: T, maxHeap: Int) -> T? {
```

Comment:
`public mutating func push(_ element: T, maxHeap: Int = Int.max) -> T? {` ?

Reply:
Well I don't want to merge the two version of `push()` since `push()` with the maxHeap has a slight performance degradation. So, I think we don't need the default parameter.

Reply:
👍